__tests__/index.tests.js

```diff
@@ -39,3 +40,63 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              expect.arrayContaining(["GET /api", "GET /api/topics","GET /api/articles",  ]))
```

Comment:
If we test like this, we will have to keep coming back to change the test with each endpoint added. How do you think we could create a test that will not need to be updated every time?